src/conn/conn_cache.c

```diff
@@ -46,6 +46,120 @@ __cache_config_abs_to_pct(
+        return (0);
+    }
+
+    /* Resetting values for reconfiguration. */
+    WT_RET(__wt_config_gets(session, cfg, "eviction_target", &cval));
+    cache->eviction_target = (double)cval.val;
+
+    WT_RET(__wt_config_gets(session, cfg, "eviction_trigger", &cval));
+    cache->eviction_trigger = (double)cval.val;
+
+    WT_RET(__wt_config_gets(session, cfg, "eviction_dirty_target", &cval));
+    cache->eviction_dirty_target = (double)cval.val;
+
+    WT_RET(__wt_config_gets(session, cfg, "eviction_dirty_trigger", &cval));
+    cache->eviction_dirty_trigger = (double)cval.val;
+
+    WT_RET(__wt_config_gets(session, cfg, "eviction_updates_target", &cval));
+    cache->eviction_updates_target = (double)cval.val;
+
+    WT_RET(__wt_config_gets(session, cfg, "eviction_updates_trigger", &cval));
+    cache->eviction_updates_trigger = (double)cval.val;
+
+    WT_RET(__wt_config_gets(session, cfg, "eviction_checkpoint_target", &cval));
+    cache->eviction_checkpoint_target = (double)cval.val;
+
+    /* Check for invalid configurations and automatically fix them to suitable values. */
+    if (cache->eviction_dirty_target > cache->eviction_target) {
+        if (debug_config)
+            __wt_verbose_warning(session, WT_VERB_CONFIGURATION,
+              "config eviction_dirty_target=%f is larger than eviction_target=%f, changing "
+              "eviction_dirty_target to %f.",
+              cache->eviction_dirty_target, cache->eviction_target, cache->eviction_target);
+        cache->eviction_dirty_target = cache->eviction_target;
+    }
+
+    if (cache->eviction_checkpoint_target > 0 &&
+      cache->eviction_checkpoint_target < cache->eviction_dirty_target) {
+        if (debug_config)
+            __wt_verbose_warning(session, WT_VERB_CONFIGURATION,
+              "config eviction_checkpoint_target=%f is less than eviction_dirty_target=%f, changing "
+              "eviction_checkpoint_target to %f.",
+              cache->eviction_checkpoint_target, cache->eviction_dirty_target,
+              cache->eviction_dirty_target);
+        cache->eviction_checkpoint_target = cache->eviction_dirty_target;
+    }
+
+    if (cache->eviction_dirty_trigger > cache->eviction_trigger) {
+        if (debug_config)
+            __wt_verbose_warning(session, WT_VERB_CONFIGURATION,
+              "config eviction_dirty_trigger=%f is larger than eviction_trigger=%f, changing "
+              "eviction_dirty_trigger to %f.",
+              cache->eviction_dirty_trigger, cache->eviction_trigger, cache->eviction_trigger);
+        cache->eviction_dirty_trigger = cache->eviction_trigger;
+    }
+
+    if (cache->eviction_updates_target < DBL_EPSILON) {
+        if (debug_config)
+            __wt_verbose_warning(session, WT_VERB_CONFIGURATION,
+              "config eviction_updates_target=%f is less than DBL_EPSILON=%f,changing "
+              "eviction_updates_target to %f.",
```

Comment:
`DBL_EPSILON` is an internal variable name. We should avoid displaying it to the end user:
```suggestion
              "config eviction_updates_target (%f) cannot be zero. Setting to 50% of eviction_updates_target (%f)",
```

Reply:
Comment rephrased in: [9e4baa7](https://github.com/wiredtiger/wiredtiger/pull/10548/commits/9e4baa7696ed61b4507e4766121155606277fcd8)